pyuvsim/mpi.py

```diff
@@ -75,7 +75,7 @@ def shared_mem_bcast(arr, root=0):
     itemsize = 0
     dtype = None
     Nitems = 0
-    shape = tuple()
+    shape = tuple()  # noqa
```

Comment:
what's wrong with doing this?

Reply:
This is flagged as unnecessary by the `flake8-comprehensions` plugin, which wants me to use `shape = ()` to initialize a tuple. I found that less readable so added the `# noqa` to stop it complaining at me.

Reply:
ech. FWIW I find `shape = ()` perfectly fine (and less annoying that havine a `# noqa` appended). But this is fine.